docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -19,12 +20,15 @@ CI/CD user Jenkins and a security scanner.
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/3. Create a CI/CD user
+## Step 1: Create a CI/CD user and corresponding role
```

Comment:
The `1/n` process is to show user the overall amount of steps (small) required to complete the section.

Reply:
Yeah I see that,I removed it thinking it would be easy to get out of date if steps are added or removed. (As evidenced by the fact that we had multiple "Step 3/3" before this PR)

The table of contents on the right hand side of the doc page will still show 3 steps, so I think that shows that it's a small number of steps, but happy to revert if that's what you prefer. Let me know.